lib/p2p/Pool.ts

```diff
@@ -314,9 +314,17 @@ class Pool extends EventEmitter {
     // TODO: send only to peers which accepts the pairId
   }
 
-  public broadcastOrderInvalidation = (order: OrderIdentifier) => {
+  /**
+   * Broadcasts an [[OrderInvalidationPacket]] to all currently connected peers.
+   * @param nodeToExclude the node pub key of a node to exclude from the packet broadcast
+   */
+  public broadcastOrderInvalidation = (order: OrderIdentifier, nodeToExclude?: string) => {
     const orderInvalidationPacket = new packets.OrderInvalidationPacket(order);
-    this.peers.forEach(peer => peer.sendPacket(orderInvalidationPacket));
+    this.peers.forEach((peer) => {
+      if (nodeToExclude && peer.nodePubKey !== nodeToExclude) {
+        peer.sendPacket(orderInvalidationPacket);
```

Comment:
It looks like that if `nodeToExclude` isn't specified, you won't send the packet to any of the peers

Reply:
Thanks for catching, silly mistake. I'd only tested this with 2 parties where the other party is always the taker.